content/en/docs/components/pipelines/sdk/component-development.md

```diff
@@ -92,296 +100,459 @@ def do_work(input1_file, output1_file, param1):
+    implementation:
+      container:
+        image: gcr.io/my-org/my-image@sha256:a172..752f
+        # command is a list of strings (command-line arguments). 
+        # The YAML language has two syntaxes for lists and you can use either of them. 
+        # Here we use the "flow syntax" - comma-separated strings inside square brackets.
+        command: [
+          python3, 
+          # Path of the program inside the container
+          /pipelines/component/src/program.py,
+          --input1-path,
+          {inputPath: Input 1},
+          --param1, 
+          {inputValue: Parameter 1},
+          --output1-path, 
+          {outputPath: Output 1},
+        ]
+    ```
+
+    The `command` is formatted as a list of strings. Each string in the
+    `command` is a command-line argument or a placeholder. At runtime,
+    placeholders are replaced with an input or output. In the preceding
+    example, two inputs and one output path are passed into a Python script at
+    `/pipelines/component/src/program.py`.
+
+    There are three types of input/output placeholders:
+
+    *   `{inputValue: <input-name>}`:
+        This placeholder is replaced with the value of the specified input.
+        This is useful for small pieces of input data, such as numbers or small
+        strings.
+
+    *   `{inputPath: <input-name>}`:
+        This placeholder is replaced with the path to this input as a file.
+        Your component can read the content's of that input at that path during
+        the pipeline run.
+
+    *   `{outputPath: <output-name>}`:
+        This placeholder is replaced with the path where your program writes
+        this output's data. This lets the Kubeflow Pipelines system read the
+        contents of the file and store it as the value of the specified output.
+ 
+    The `<input-name>` name must match the name of an input in the `inputs`
+    section of your component specification. The `<output-name>` name must
+    match the name of an output in the `outputs` section of your component
+    specification.  
+
+### Define your component's interface
+
+The following examples demonstrate how to specify your component's interface. 
+
+1.  To define an input in your `component.yaml`, add an item to the
+    `inputs` list with the following attributes:
+
+    *   `name`: Human-readable name of this input. Each input's name must be
+        unique.
+    *   `description`: (Optional.) Human-readable description of the input.
+    *   `default`: (Optional.) Specifies the default value for this input.
+    *   `type`: (Optional.) Specifies the input’s type. Learn more about the
+        [types defined in the Kubeflow Pipelines SDK][dsl-types] and [how type
+        checking works in pipelines and components][dsl-type-checking].
+    *   `optional`: Specifies if this input is optional. The value of this
+        attribute is of type `Bool`, and defaults to **False**.
+
+    In this example, the Python program has two inputs: 
```

Comment:
Suggest changing "In this example," to "In the following example,"

Reply:
I prefer to use "In this example" since this doc is one example discussed in several sections. WDYT?